modules/gapi/include/opencv2/gapi/gstreaming.hpp

```diff
@@ -371,6 +371,29 @@ class GAPI_EXPORTS_W_SIMPLE GStreamingCompiled
 };
 /** @} */
 
+namespace gapi {
+/**
+ * @brief Ask G-API to set specific queue capacity for streaming execution.
+ *
+ * For streaming execution G-API has a special concurrent bounded queue to
+ * fetch next frame while current is being processed. This compilation argument
+ * specifies the capacity of this queue.
+ */
+struct queue_capacity
```

Comment:
OpenCV uses `CamelCase` for structures/classes/enum types.

Reply:
yes and so we do, but for some reason the graph compile arguments were started as `snake_case` (probably, unintentionally) so we follow it there now.

It is worth aligning in scope of 5.0 I believe.